### PR TITLE
CU-2akhvqp Add README and Apache License

### PR DESCRIPTION
**Pull request by andymiller813**

### CODE REVIEW

**Comment by newmskywalker** [comment]
Task linked: [CU-2akhvqp Update repo README for community contribution guidelines](https://app.clickup.com/t/2akhvqp)

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (cristhianescobar) [inline LICENSE.txt]: Should we replace this with `Copyright 2022 Project Newm`

File: README.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+*** See the bottom of this document for the declaration of the reference variables
+*** for contributors-url, forks-url, etc. This is an optional, concise syntax you may use.
+*** https://www.markdownguide.org/basic-syntax/#reference-style-links
+-->
+
+[![Contributors][contributors-shield]][contributors-url]
+[![Forks][forks-shield]][forks-url]
+[![Stargazers][stars-shield]][stars-url]
+[![Issues][issues-shield]][issues-url]
+[![MIT License][license-shield]][license-url]
```
Comment (cristhianescobar) [inline README.md]: Are we also doing `MIT License` ?

File: LICENSE.txt (hunk 1 above)
Comment (andymiller813) [reply LICENSE.txt]: Ha, I looked at a Square repo and they still had it like this so I left it. 🤷 
We probably should though.  I updated our Copyright in the readme

File: README.md (hunk 2 above)
Comment (andymiller813) [reply README.md]: Nah, updated

**Review by cristhianescobar — APPROVED** [review]
🦖

**Pull request merged**